From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 18 Jul 2016 23:50:30 +0200
Subject: [PATCH] build/Makefile: Overrideable product, commit hash.

For Armband's build system, using only the Fuel commit ID is not
enough to fully describe the state of the source code, as patches
are also applied to other fuel modules (as git submodules).
Instead, a pointer to a valid Armband commit ID should be used.

While at it, allow product name to be overriden, to signal the
commit ID should be looked up inside the Armband repository.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/Makefile b/build/Makefile
index ec5d27a..ba117da 100644
--- a/build/Makefile
+++ b/build/Makefile
@@ -21,11 +21,11 @@ SHELL = /bin/bash
 export MOSVERSION = 9.0
 export ISOSRC = file:$(shell pwd)/fuel-$(MOSVERSION).iso
 export ISOCACHE = $(shell pwd)/$(shell basename $(ISOSRC))
-export PRODNO = "OPNFV_FUEL"
+export PRODNO ?= "OPNFV_FUEL"
 export REVSTATE = "P0000"
 export USER ?= $(shell whoami)
 export BUILD_DATE = $(shell date --utc +%Y-%m-%d:%H:%M)
-export OPNFV_GIT_SHA = $(shell git rev-parse HEAD)
+export OPNFV_GIT_SHA ?= $(shell git rev-parse HEAD)
 # Store in /etc/fuel_build_id on fuel master
 export BUILD_ID := $(PRODNO)_$(BUILD_DATE)_$(OPNFV_GIT_SHA)
 
